src/napari/_app_model/context/_layerlist_context.py

```diff
@@ -180,6 +180,10 @@ def _empty_shapes_layer_selected(s: LayerSel) -> Callable[[], bool]:
     return check_fun
 
 
+def _active_has_features(s: LayerSel) -> bool:
+    return hasattr(s.active, 'features')
```

Comment:
Should it return true if layer have empty? Current implementation does not allow adding column to data

Reply:
I don't understand how these two sentences are related?